include/wx/webview_chromium.h

```diff
@@ -0,0 +1,323 @@
+/////////////////////////////////////////////////////////////////////////////
+// Author: Steven Lamerton
+// Copyright: (c) 2013 Steven Lamerton
+// Licence: wxWindows licence
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef _WX_WEBVIEWCHROMIUM_H_
+#define _WX_WEBVIEWCHROMIUM_H_
+
+#include "wx/defs.h"
+
+#if wxUSE_WEBVIEW && wxUSE_WEBVIEW_CHROMIUM
+
+#include "wx/control.h"
+#include "wx/webview.h"
+#include "wx/sharedptr.h"
+#include "wx/vector.h"
+
+#ifdef __VISUALC__
+#pragma warning(push)
+#pragma warning(disable:4100)
+#endif
+
+#include "include/cef_browser.h"
```

Comment:
This would probably represent a significant change, but it would be great if we could avoid including all this CEF stuff from this public header. AFAICS the only thing actually needed here is `m_clientHandler` and we could just wrap it into some `wxWebViewChromiumImpl` to avoid having it here.

There would be quite a few advantages of keeping this header free of CEF stuff, in particular we could even allow people to use this backend without installing CEF headers at all if we just build the official binaries with it, it would be enough to distribute just the DLLs.